pkg/app/redirect/validator_test.go

```diff
@@ -96,7 +101,14 @@ var _ = Describe("Validator suite", func() {
 			Entry("Quad Tab 2", "/\t\t\\\t\t/evil.com", false),
 			Entry("Relative Path", "/./\\evil.com", false),
 			Entry("Relative Subpath", "/./../../\\evil.com", false),
-			Entry("Partial Subdomain", "evilbar.foo", false),
+			Entry("Valid Wildcard Subdomain", "foo.wildcard.bar/redirect", true),
+			Entry("Valid Wildcard Subdomain Root", "wildcard.bar/redirect", true),
+			Entry("Valid Wildcard Subdomain anyport", "foo.wildcard.sub.anyport.bar:4242/redirect", true),
+			Entry("Valid Wildcard Subdomain Anyport Root", "wildcard.sub.anyport.bar:4242/redirect", true),
+			Entry("Valid Wildcard Subdomain Defined Port", "foo.wildcard.sub.port.bar:8080/redirect", true),
+			Entry("Valid Wildcard Subdomain Defined Port Root", "wildcard.sub.port.bar:8080/redirect", true),
+			Entry("Invalid HTTP subdomain", "foo.bar/redirect", false),
+			Entry("Invalid HTTP subdomain", "proxy.foo.bar/redirect", false),
```

Comment:
Are we sure about this? I don't think redirects are allowed without a scheme prefixing them currently, this could lead to open redirects which I really would like to avoid

Reply:
Thanks, forget to commit the fix.
Done.